packages/mdc-radio/mdc-radio.scss

```diff
@@ -47,14 +47,6 @@ $mdc-radio-transition-duration: 120ms;
   cursor: pointer;
   will-change: opacity, transform, border-color, background-color, color;
 
-  &::before,
```

Comment:
I believe this is replaced by setting radius in ripple-config.

Reply:
@kfranqueiro helpfully pointed out in a previous PR that we can achieve the same result by passing `radius: 50%` to the `mdc-ripple-bg` and `mdc-ripple-fg` mixins, so now I'm doing that in the `mdc-radio-ripple` mixin instead of hard-coding here.